apps/errors/errors.go

```diff
@@ -75,3 +76,19 @@ func (e CallErr) Verbose() string {
 	e.Resp.TLS = nil     // Same
 	return fmt.Sprintf("%s:\nRequest:\n%s\nResponse:\n%s", e.Err, prettyConf.Sprint(e.Req), prettyConf.Sprint(e.Resp))
 }
+
+// Unwrap returns the wrapped error
+func (e CallErr) Unwrap() error {
+	return e.Err
+}
+
+// Is reports whether any error in errors chain matches target.
+func Is(err, target error) bool {
```

Comment:
I don't think we need this func or the As() func below.  They just appear to redefine the ones from the standard lib.

Reply:
The reason it was added is , if we are trying to use this like its used here : https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/178/files#diff-a92c747c9d32f8b8080b85c19048958628dae0c99f59cc4f92e0262572ffa4f4R203 , we will need two imports , one is the standard errors package and the other one will be ours exposing CallError and we will have to name either one to differentiate. I believe customers trying to do this would also face similar issues. So if we can expose it as a part of our errors package, it will be easier

Reply:
Oh yes I remember this discussion now.